packages/extension/src/ui/features/accounts/upgrade.service.ts

```diff
@@ -219,11 +224,18 @@ export const useCheckUpgradeAvailable = (account?: Account) => {
     data: needsUpgrade,
     error: needsUpgradeError,
     isValidating: needsUpgradeValidating,
+    mutate,
   } = useSWR(
     [accountIdentifier, accountClassHash, "showUpgradeBanner"],
-    () => account && checkIfUpgradeAvailable(account, accountClassHash),
-    { suspense: false },
+    () => {
+      if (!account) {
+        return false
+      }
+
+      return checkIfUpgradeAvailable(account, accountClassHash)
+    },
+    { suspense: false, ...withPolling(60 * 1000) },
```

Comment:
`suspense` defaults false

```suggestion
    { ...withPolling(60 * 1000) },
```